widget/embedded/src/index.ts

```diff
@@ -126,3 +127,5 @@ export {
   PendingSwapNetworkStatus,
   EventSeverity,
 };
+// Internal constant exports for Rango
```

Comment:
It's not internal, it can be used by anyone.

Reply:
Changed to "Constant exports for Rango"

Reply:
You don't need to export in a separate place. Just add it next to `StepExecutionBlockedEventStatus`

Reply:
Done